### ISSUE TITLE
kvantum can't be used a theme for QT apps anymore

### ISSUE THREAD

**Issue by Wolfyxon** [tsujan/Kvantum#952]
After updating, QT apps can't use kvantum as a theme anymore.
```
QApplication: invalid style override 'kvantum' passed, ignoring it.
	Available styles: Windows, Fusion
```
```
qrc:/qml/kdenliveclipmonitor.qml: module "kvantum" is not installed
Segmentation fault (core dumped)
```

`kvantummanager` and `kvantumpreview` work properly and show the correct theme.

I installed kvantum from the Arch Linux repos.

**Comment by tsujan** [comment]
> I installed kvantum from the Arch Linux repos

That package is only for Qt6, while your app should be in Qt5. Install kvantum-qt5 for Qt5 apps.

**Comment by Wolfyxon** [comment]
Yeah that worked, thanks. I guess the package was split into 2 versions suddenly.

**Comment by tsujan** [comment]
> I guess the package was split into 2 versions suddenly.

I also think so. Why they did it, I don't know; it wasn't needed.

Anyway, when I release V1.1.0 (after a stable version of KF6 comes to important distros), some things about compilation will change — as is already the case with git Kvantum.
